charts/core/src/Echart/Echart.types.ts

```diff
@@ -0,0 +1,99 @@
+import { Theme } from '@leafygreen-ui/lib';
+
+import type { ChartOptions, SeriesOption } from '../Chart';
+
+type EChartsEvents =
```

Comment:
Should this be a enum? Not necessary, just wanted to call it out

Reply:
Yeah, why not. Will update (probably with the help of my copilot 😄)